tests/test_core.py

```diff
@@ -1291,7 +1293,7 @@ def test_exceptions_allowed_uncaught(self):
     # mismatches are fixed in fixImports() function in JS glue code.
     self.set_setting('EXCEPTION_CATCHING_ALLOWED', ["_Z4testv"])
     # otherwise it is inlined and not identified
-    self.set_setting('INLINING_LIMIT')
+    self.set_setting('INLINING_LIMIT', 50)
```

Comment:
This too. I created this function in #10720 by copy-pasting `test_exceptions_allowed_2`, which already had the incorrect line.